configs/11.0/packages/gcc/stage_1

```diff
@@ -26,30 +26,28 @@ ATCFG_BUILD_STAGE_T='dir'
 
 # Required pre build hacks
 atcfg_pre_hacks() {
+	# IBM maintains branches on the svn repository for GCC.
+	# Check we are using an IBM branch. If not, stop everything.
+	if [[ ! -s ${ATSRC_PACKAGE_WORK}/gcc/REVISION ]]; then
+		echo "ERROR: gcc/REVISION is missing and/or empty"
+		exit 1
+	fi
+	if ! grep -Eqs "^\[ibm/gcc-[0-9]+-branch revision [0-9]+]$" ${ATSRC_PACKAGE_WORK}/gcc/REVISION; then
+		echo "ERROR: Not using an IBM branch"
+		exit 1
+	fi
```

Comment:
OK.  I like this check better than what we have today (the comparison between `REVISION.back` and the `sed` output)